### ISSUE TITLE
Load historical presence data on demand

### ISSUE THREAD

**Issue by ara4n** [vector-im/vector-web#1295]
Especially for offline users, load last active data on demand?

**Comment by mebjas** [comment]
@ara4n are you suggesting being able to open `vector.im` offline with last cached messages / rooms ?

**Comment by ara4n** [comment]
no, i'm suggesting that when one hovers over a memberlist, it should explicitly query the presence API for that user if we don't already have presence data for them, to lazy-load it in on demand.

**Comment by mebjas** [comment]
:D that's in a whole different galaxy than I though it'd be. Wise to discuss what I though it is, in another issue?

**Comment by reicheltmediadesign** [comment]
This is still a thing. I just had multiple users in my contacts shown as "online" (green dot), even if they are offline. This happens on Windows (Element Desktop) and on my iOS App, but NOT on the Web version. After cleaning the cache, the user is showed as offline as it should be.

**Comment by flyinghuman** [comment]
we also have Problems with the Presence Status on different Clients (e.g. Win Desktop App + mobile App on Android/iOS). Some Clients show Online for Users which where definitive Offline and other Client (WebApp) show the correct Status (Offline). Clearing the Cache solves the Problem - but this cannot be a solution! It MUST be that Element fetches periodically (or via PUSH?) the actual Precense Status for Users so that you can be sure that the User is really Online or Offline. 
Please review this again and give it a higher priority! Thanks!
